src/imagings/requests/NewImagingRequest.tsx

```diff
@@ -46,10 +48,21 @@ const NewImagingRequest = () => {
   useAddBreadcrumbs(breadcrumbs)
 
   const onPatientChange = (patient: Patient) => {
-    setNewImagingRequest((previousNewImagingRequest) => ({
-      ...previousNewImagingRequest,
-      patient: patient.fullName as string,
-    }))
+    if (patient) {
+      setNewImagingRequest((previousNewImagingRequest) => ({
+        ...previousNewImagingRequest,
+        patient: patient.fullName as string,
```

Comment:
If this store as patient id, the `patient` field on the table in requested imagings page would display its patient id instead of full name
![image](https://user-images.githubusercontent.com/40738727/90529570-db80da00-e141-11ea-920d-d00f3252f994.png)